Controller/AuthorizeController.php

```diff
@@ -26,8 +26,10 @@
  *
  * @author Chris Jones <leeked@gmail.com>
  */
-class AuthorizeController extends ContainerAware
+class AuthorizeController
 {
+    use ContainerAwareTrait;
```

Comment:
Same as per php 5.4... it's not supported any longer. Do you want to burden yourself with it?